### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by rossf7**
After this PR we can create the first release. Next up I'd like to use it in app-exporter so we can have some tests there.

### CODE REVIEW

File: integration/setup/setup.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import (
+	"os"
+	"testing"
+)
+
+func Setup(m *testing.M, config Config) {
+	var v int
+
+	// Add any setup tasks that should execute for all tests here.
```
Comment (rossf7) [inline integration/setup/setup.go:13]: In this case there aren't any setup tasks.

File: integration/test/bootstrap/bootstrap_test.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	"github.com/giantswarm/helmclient/v2/pkg/helmclient"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/apptestctl/integration/key"
+)
+
+// TestBootstrap ensures that the chartmuseum app CR is deployed. This confirms
+// that the app platform is installed and can deploy app CRs.
+//
```
Comment (rossf7) [inline integration/test/bootstrap/bootstrap_test.go:21]: The test is pretty simple and we need to create the chartmuseum app CR anyway.

File: integration/test/bootstrap/setup.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd project/
+go install .
+
+# Start the bootstrap process.
+apptestctl bootstrap --kubeconfig="$(kind get kubeconfig)"
```
Comment (rossf7) [inline integration/test/bootstrap/setup.sh:7]: This script configured via the integration test job installs the CLI and starts the bootstrap.

Comment (tomahawk28) [reply integration/test/bootstrap/setup.sh:7]: 👍

**Review by tomahawk28 — APPROVED** [review]
LGTM

**Pull request merged**